### PR TITLE
Allow RDI reader to skip junk headers

### PR DESCRIPTION
**Pull request by jmcvey3**
Properly searches for and skips 7f79 header IDs.

Solution for https://github.com/lkilcher/dolfyn/issues/128

### CODE REVIEW

**Comment by jmcvey3** [comment]
Hi @ivicajan, see if this works for you as well.

**Comment by jmcvey3** [comment]
@ssolson Hopefully this is the last new one in the dolfyn pile of PRs

**Comment by jmcvey3** [comment]
Just need to add test data and this one should be good to go

File: mhkit/dolfyn/io/rdi.py [hunk 1]
```diff
@@ -771,16 +771,16 @@ def read_cfgseg(self, bb=False):
         n_cells = fd.read_ui8(1)
         if ("n_cells" not in cfg) or (n_cells != cfg["n_cells"]):
             cfg["n_cells"] = n_cells
-            if self._debug_level >= 1:
-                logging.warning(f"Number of cells set to {cfg['n_cells']}")
+            if self._debug_level > 0:
+                logging.info(f"Number of cells set to {cfg['n_cells']}")
```
[the remark this answers is not in the document]
Comment (jmcvey3) [reply mhkit/dolfyn/io/rdi.py:775]: Ah, no it's completely subjective. 0 is a high-level overview of what's happening, 1 shows what functions the code is running through, and 2 shows a low level task by task breakdown.

[the remark this answers is not in the document]
Comment (ssolson) [reply mhkit/dolfyn/io/rdi.py:775]: Thanks James. I'm ready to merge this if you are.

**Pull request merged**